coaster/examples/readme.rs

```diff
@@ -1,27 +1,47 @@
 extern crate coaster as co;
 extern crate coaster_nn as nn;
-use crate::co::prelude::*;
-use crate::nn::*;
+
+use co::prelude::*;
+use co::frameworks::native::flatbox::FlatBox;
+use nn::*;
 
 fn write_to_memory<T: Copy>(mem: &mut FlatBox, data: &[T]) {
-	let mem_buffer = mem.as_mut_slice::<T>();
-	for (index, datum) in data.iter().enumerate() {
-	    mem_buffer[index] = *datum;
-	}
+    let mem_buffer = mem.as_mut_slice::<T>();
+    for (index, datum) in data.iter().enumerate() {
+        mem_buffer[index] = *datum;
+    }
 }
 
+#[cfg(not(feature = "cuda"))]
 fn main() {
-    // Initialize a CUDA Backend.
     let backend = Backend::<Native>::default().unwrap();
     // Initialize two SharedTensors.
     let mut x = SharedTensor::<f32>::new(&(1, 1, 3));
     let mut result = SharedTensor::<f32>::new(&(1, 1, 3));
+    let payload: &[f32] = &::std::iter::repeat(1f32).take(x.capacity()).collect::<Vec<f32>>();
+    let native = Backend::<Native>::default().unwrap();
+    write_to_memory(x.write_only(native.device()).unwrap(), payload); // Write to native host memory.
+    // FIXME: Add in Sigmoid example for native
```

Comment:
I think there is a little something missing here :)

Reply:
That was my comment in chat yesterday, I have no idea how to get it to work here! You're welcome to add in a solution if you know it :)

Reply:
Ahh yes, ofc, I'll take a look into it :)